Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -262,15 +264,18 @@ \subsection{Milestones}
 
 \subsubsection{Design Phase}
 \label{subsubsec: design phase}
-    This will be the first phase and reappearing after the culmination of the experimentation phase.
-    Goals are to design the propulsion system, re-design the test stand, design the minimum length and aerospike nozzle
+    This will be the first phase of the project.
+    The goals are to design the propulsion system, design the test stand, design the minimum length supersonic nozzle. At the end of this phase
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
+    will use it as a guide (inlcuding drawings for machinists).
```

Comment:
`This will be the first phase of the project.` can be removed, because it is being described first